pom.xml

```diff
@@ -3,9 +3,9 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>life.qbic</groupId>
-	<artifactId>templateQbicPortlet</artifactId>
+	<artifactId>qbic_portlet_template</artifactId>
```

Comment:
I would not use '_' but '-' like: `qbic-portlet-template`

And qbic is quite redundant in the artifactID, as it is already in the groupID.
So I would go for: `portlet-base` or so